scripts/crowbar-testbuild.py

```diff
@@ -44,6 +44,7 @@
 }
 
 MKCLOUD_HA_PARAMETERS = (
+    'label=openstack-mkcloud-ha-x86_64',
```

Comment:
@bmwiedemann not right now, but we might soonish for s390x. I'd say we can always update that later.